lib/utils/response-header.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+var mime = require('mime');
+var url = require('url');
+var fs = require('fs');
+var path = require('path');
+
+var handlebars = require('handlebars');
+var dirTemplate = handlebars.compile(fs.readFileSync(path.resolve(__dirname, '..', 'templates/dir.html')).toString());
+
+
+function updateFilenameHeader(req, filename) {
+  var broccoliInfo = req.headers['x-broccoli'];
+  if (broccoliInfo) {
+    broccoliInfo.filename = filename;
+    req.headers['x-broccoli'] = broccoliInfo;
```

Comment:
This line seems to be a no-op; if the above conditional matches then `req.headers['x-broccoli'] == broccoliInfo` already